src/d3a/models/strategy/smart_meter.py

```diff
@@ -18,6 +18,8 @@
 
 from d3a_interface.constants_limits import ConstSettings
 from d3a_interface.validators.smart_meter_validator import SmartMeterValidator
+from d3a_interface.enums import SpotMarketTypeEnum
+from d3a_interface.validators.home_meter_validator import HomeMeterValidator
```

Comment:
This seems to be the incorrect output of a merge conflict. `HomeMeterValidator` doesn't exist anymore (`SmartMeterValidator` is used and it's already imported above)